scala-steward.conf

```diff
@@ -31,6 +31,11 @@ updates.ignore = [
   # See https://www.scala-lang.org/blog/2022/08/17/long-term-compatibility-plans.html#owners-of-commercial-projects
   {groupId = "org.scala-lang", artifactId = "scala3-library", version = {prefix = "3.4."}},
   {groupId = "org.scala-lang", artifactId = "scala3-library", version = {prefix = "3.5."}},
+
+  # Ignore updates to Apache Thrift and Scrooge Thrift.
+  # See https://trello.com/c/1rp2MVVA and https://github.com/guardian/content-api/issues/2903
+  {groupId = "org.apache.thrift", artifactId = "libthrift", version = {prefix = "0.20."}},
+  {groupId = "com.twitter", artifactId = "scrooge-core", version = {prefix = "22.12."}},
```

Comment:
I think every version from [21.9.0](https://github.com/twitter/scrooge/releases/tag/scrooge-21.9.0) forward is broken: will we need a change here to make sure scala steward doesn’t attempt to update to a version at or above 21.9.0 and below 22.12.0?

Reply:
It depends whether we want to allow warnings or not. 21.9.0 added the reserved keywords to ThriftParser but it only throws warnings. 

<img width="859" alt="image" src="https://github.com/user-attachments/assets/2a8dcb77-ec33-44b1-b162-96367e61078e">

It looks like from [22.2.0](https://github.com/twitter/scrooge/releases/tag/scrooge-22.2.0) ThriftParser will throw exceptions.